client/jetpack-connect/auth-form-header.jsx

```diff
@@ -166,7 +167,7 @@ export class AuthFormHeader extends Component {
 			const pluginNames = {
 				'jetpack-ai': 'Jetpack AI',
 				'jetpack-boost': 'Jetpack Boost',
-				default: 'Jetpack',
+				default: 'Jetpack and WooPayments',
```

Comment:
This should be dynamic depending on the extensions chosen. We should also make it customizable for other extensions as well. For now, let's go with hard coded text as we're not merging this PR until we have other changes are done.

Reply:
should it be translated?